wiki/Game_mode/osu!taiko/en.md

```diff
@@ -24,24 +24,22 @@ Alternatively, click on the `Mode` button and select `osu!taiko`.
 
 For players with prior *Taiko no Tatsujin* experience:
 
-- No chibi dancers at the bottom (must be storyboarded)
-- The health bar must be at least 50% full to pass the song (which is not really obvious using the default skin; please look in the [Skin Compendium](https://osu.ppy.sh/community/forums/topics/180864) for a decent osu!taiko skin)
+- No PuchiChara at the bottom (must be storyboarded)
```

Comment:
the change makes it far harder to understand what it is, please revert (also, IMO the specific name isn't important here in context of the game)

Reply:
no, the dancers bit